app/controllers/oidc/user_info_controller.rb

```diff
@@ -2,25 +2,11 @@ class Oidc::UserInfoController < ApplicationController
   before_action :authenticate_token!
 
   def show
-    oidc_response = { sub: pairwise_subject_identifier }
+    oidc_response = { sub: @token[:pairwise_subject_identifier] }
 
     render json: Claim
-             .where(subject_identifier: subject_identifier)
-             .select { |c| Permissions.any_of_scopes_can_read(c.claim_name, token_scopes) }
+             .where(subject_identifier: @token[:true_subject_identifier])
+             .select { |c| can_read? c.claim_name }
```

Comment:
Interesting Rubocop doesn't kick off about this one, Iwonder if it's because it's within a select 🤔 ?
Isn't there a preference to use parenthesis to wrap method parameters?

Reply:
I think the preference is only for if the code is ambiguous, not in general.  We have parentheses-less code all over the place, eg all the `head :status` bits